posts/2024-03-29-ArgoCD-Configuration-Drift.adoc

```diff
@@ -0,0 +1,161 @@
+image::/img/blog/configuration-drift-3.png[Creating the App,width=70%,align="center"]
+
+Select the configiration options that best suit your application. For the purposes of just trying out and getting started with ArgoCD, feel free to copy the configuration we've set in our example app.
+
+Next, you will need to scroll down to the configuration section labelled "Source", shown in the image below. Here, you will need to set several configuration options.
+
+First, change the Repository URL to your newly forked repo. This tells ArgoCD what git repository has the configuration for the application you are looking to deploy. 
+
+You'll also need to change the path, which is the path in the git repo where ArgoCD is looking for deployment files to deploy. 
+
+Additionally, you'll need to set the Cluster URL. In this example, we have set it to https://kubernetes.default.svc, which is the local cluster URL.
+
+Then, lastly, you will need to set the namespace. In our case, this is daytrader7 (which will be created because we selected the "Auto-Create Namespace" option).
+
+
+After verifying these options, click Create in the top left.  This creates an "Application" resource that ArgoCD uses to watch the repository we gave it.  
+
+[.img_border_light]
+image::/img/blog/configuration-drift-4.png[Setting the Repo,width=70%,align="center"]
+
+[#manually_sync]
+=== Manually Syncing your Application with the Repository
+
+Since we chose manual sync policy in the options for demonstrative purposes, we have to sync the application manually for ArgoCD to deploy the application. If you have also selected this option, on the main page of the ArgoCD console, you should see a new tile that looks like this:
+
+[.img_border_light]
+image::/img/blog/configuration-drift-5.png[Argo Dashboard,width=70%,align="center"]
+
+Click the Sync button. Then, on the window that appears, click Sync again.
+To monitor the progress of this application sync, you can click on the daytrader7 tile.
+Over the course of a few minutes, you should see resources being created and the app dashboard should look like this:
+[.img_border_light]
+image::/img/blog/configuration-drift-6.png[App deploying,width=70%,align="center"]
+
+This diagram shows the many kubernetes resources created for the application, such as the deployment, pod, and service.  We can also monitor the health of each of these resources and verify they were created properly.  
+
+
+Now that the application is deployed and all the resources are created we need to make sure it is exposed externally.  This sample sets `.spec.expose` to `true`, which automatically creates an external route on OpenShift.  If you are on Kubernetes, you need to either configure a load balancer or an Ingress controller for the application.  Find the URL via `oc get routes –n daytrader7` if on OpenShift, or `kubectl get ingress -n daytrader7` if on Kubernetes with Ingress configured.  Then paste the URL in a browser to access the Daytrader login page.
```

Comment:
> This sample sets `.spec.expose` to `true`

Where?

I'd break the kube/RHOS commands out onto separate lines to make them clearer